### ISSUE TITLE
Not able to complete forms

### ISSUE THREAD

**Issue by flopich** [pluginsGLPI/formcreator#3520]
**Describe the bug**
Since updating the plugin, in the debug mode we noticed that the tablespace entityconfigs was missing from the Database which results in not being able to interact with forms for Self-service users and not being able to complete them even as a super-admin.

**Screenshots**
Can't provide screenshots for corporate security reasons

**GLPI / Plugins (please complete the following information):**
- GLPI version : 10.0.15
- list of all plugins with their version : Formcreator, Additionnal fields & Data injection

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome

**Additional context**
The plugin was on a RC update (2.13.0-rc1) before updating to the latest version (2.13.9)

**Comment by btry** [comment]
Hi

a  tablespace ? Do you mean the table glpi_plugin_formcreator_entityconfigs ?

**Comment by flopich** [comment]
> Hi
> 
> a tablespace ? Do you mean the table glpi_plugin_formcreator_entityconfigs ?

Yes, we noticed that we have another table missing alongside this one : "glpi_plugin_formcreator_issues"
For info, we updated the plugin this way : 

- Deactivate the former version of the plugin
- Put the new version of the plugin in 10.0.15/plugins by removing the old one first
- Activate the plugin after noticing a request to update the plugin on the Plugins page of GLPI

**Comment by btry** [comment]
the table glpi_plugin_formcreator_issues should not be deleted. 

Run bin/console db:check -p formcreator and show the result

**Comment by flopich** [comment]
This is the output that we have : SQL error "2006": MySQL server has gone away in this query " SHOW CREATE TABLE 'glpi-plugin_formcreator_entityconfigs'"
Unable to get table "glpi_plugin_formcreator_entityconfigs"

**Comment by btry** [comment]
You have a problem unrelated to Formcreator, but which WILL break your GLPI and cause DB inconsistencies. 

Check your server in deep : free space on volumes, maybe hardware reliareliability ...

**Comment by flopich** [comment]
I was able to rollback to the previous version of the plugin upon uninstall and reinstall of the RC version I had, everything seems fixed now.

**Comment by flopich** [comment]
Truth is, we did notice some issues in the past (2 months ago I'd say) where the MySQL server would be constantly at 100% upon specific ticket filtering queries. 

Something that we also noticed when trying to find the answer to our issue is that when we : 

1. Exported the forms list in JSON through the plugin (front/form.php) button at the top
2. Uninstalled and clean installed the plugin again
3. Imported the forms from the JSON file generated by the plugin

The MySQL server would crash upon import of the file. We were stuck at 1/190 objects imported (even though we only have 6 forms). Happened every single time.
